addons/sale/views/sale_order_views.xml

```diff
@@ -879,6 +879,7 @@
         <field name="inherit_id" ref="sale.view_sales_order_filter"/>
         <field name="arch" type="xml">
             <filter name="my_sale_orders_filter" position="after">
+                <filter invisible="1" string="Sales Orders" name="sales" domain="[('state', 'not in', ('draft', 'sent', 'cancel'))]"/>
```

Comment:
```suggestion
                <filter invisible="1" string="Sales Orders" name="sales" domain="[('state', '=', 'sale')]"/>
```
Now that the locked (done) state has been moved to a specific field, we can simplify the domain ;)